### PR TITLE
hint for both faculty and department in the free search && Initial Snack Bar

### PR DESCRIPTION
**Pull request by Nitay880**

### CODE REVIEW

File: academic_helper/views/basic.py [hunk 1]
```diff
@@ -29,6 +29,7 @@ def get_context_data(self, **kwargs):
         log.info(f"Rendering {wrap(self.title)} for {wrap(self.user)}")
         context = super().get_context_data(**kwargs)
         context["title"] = self.title
+        context["messages"]=["I <3 Snacks"]
```
Comment (asaf-kali) [inline academic_helper/views/basic.py:32]: If I understand correctly, this will pop up on every page load?

File: templates/courses/courses.html [hunk 2]
```diff
@@ -136,6 +137,8 @@
         $(document).ready(() => {
             autocomplete(document.getElementById("faculty"), {{ all_faculties|safe }});
             autocomplete(document.getElementById("department"), {{ all_departments|safe }});
+            autocomplete(document.getElementById("free_text"), {{ all_faculties_and_departments|safe }});
```
Comment (asaf-kali) [inline templates/courses/courses.html:140]: I disagree with that.
@evyatar100 @shanishapp  what do you think?

File: templates/master.html [hunk 3]
```diff
@@ -120,5 +121,14 @@
 <div id="content-body-container" style="margin: 25px;">
     {% block content_body %} {% endblock %}
 </div>
+{% if messages %}
+  <script>
+      {% for message in messages %}
+          $(document).ready(function () {
+              $('.toast').toast('show');
+          });
+      {% endfor %}
```
Comment (asaf-kali) [inline templates/master.html:130]: Nice.
Did you test it with multiple messages?

File: academic_helper/views/courses.py [hunk 4]
```diff
@@ -52,6 +52,7 @@ def get_context_data(self, **kwargs):
         departments = Department.objects.values_list("name", flat=True)
         context["all_faculties"] = sorted(set(faculties))
         context["all_departments"] = sorted(set(d.replace(":", "").strip() for d in departments))
+        context["all_faculties_and_departments"] = sorted(context["all_departments"]+context["all_faculties"])
```
Comment (asaf-kali) [inline academic_helper/views/courses.py:55]: Im not sure this is needed.

File: templates/master.html (hunk 3 above)
Comment (Nitay880) [reply templates/master.html:130]: I tested it with 2 , it looks good, although they weren't long.

File: academic_helper/views/basic.py (hunk 1 above)
Comment (Nitay880) [reply academic_helper/views/basic.py:32]: yup, that's indeed what's hapenning.

File: templates/message/message.html [hunk 5]
```diff
@@ -1,24 +1,34 @@
...
-          <i class="fas fa-check mr-2"></i>
-        {% endif %}
-        {{message.tags|capfirst}}
-      </strong>
-      <button type="button" class="ml-2 mb-1 close" data-dismiss="toast" aria-label="Close">
-        <span aria-hidden="true">&times;</span>
-      </button>
+
+{% load static %}
+<link rel="stylesheet" href="{% static 'styles/message.css' %}">
```
Comment (Nitay880) [inline templates/message/message.html:3]: this file is only optional, it looks fantastic right now, worth checking

File: templates/message/message.html [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+<div aria-live="polite" aria-atomic="true" style="position: relative; min-height: 200px;">
+  <!-- Position it -->
+  <div style="position: fixed; top: 90px; right: 28px;z-index:2">
+
```
Comment (Nitay880) [inline templates/message/message.html:7]: image
it's pretty neat

Comment (asaf-kali) [reply templates/message/message.html:7]: This is cool, but please make sure to remove the top white-blank bar... this can't be merged like that.

**Comment by asaf-kali** [comment]
Continued with #126

**Pull request closed without merging**